library/ssl_tls.c

```diff
@@ -9961,7 +9961,8 @@ static int ssl_session_load( mbedtls_ssl_session *session,
         if( (size_t)( end - p ) < sizeof( ssl_serialized_session_header ) )
             return( MBEDTLS_ERR_SSL_BAD_INPUT_DATA );
 
-        if( mbedtls_platform_memcmp( p, ssl_serialized_session_header,
+        // use regular memcmp as session header is not that critical
+        if( memcmp( p, ssl_serialized_session_header,
```

Comment:
Also public data vs known value (constant in the source code).

Reply:
fixed